app/views/ReportView/components/ExpressionCorrelation/index.tsx

```diff
@@ -10,7 +10,7 @@ import ReportContext from '@/context/ReportContext';
 import ImageService from '@/services/reports/image.service';
 import DataTable from '@/components/DataTable';
 import { getPairwiseExpressionCorrelation } from '@/services/reports/pairwise-expression';
-import Image from '@/components/Image';
+import Image, { ImageType } from '@/components/Image';
```

Comment:
I get what this is doing, but I don't think the API is updated to give an `array` instead of `map`

![image](https://user-images.githubusercontent.com/25511396/119890836-26a7f480-beed-11eb-913d-9c06a6574547.png)